setup.py

```diff
@@ -33,6 +33,9 @@ def read_requirements(path):
       license='Apache License 2.0',
       packages=find_packages(),
       install_requires=read_requirements('requirements/main.txt'),
+      package_data={
```

Comment:
thx for this addon @Seirdy! we will merge it once we sort out our workflow CI setup